app/config/jobConfig.js

```diff
@@ -27,6 +27,7 @@ let config = {
     arrivalCount: parseInt(process.env.ARRIVAL_COUNT) || undefined,
     rampTo: parseInt(process.env.RAMP_TO) || undefined,
     maxVusers: parseInt(process.env.MAX_VIRTUAL_USERS) || undefined,
+    // WHY DO WE NEED EMAILS AND WEBHOOKS?
```

Comment:
lets leave this for now, we have an issue: https://github.com/Zooz/predator/issues/390

that should remove all of the unnecessary env variables and should retrieve needed parameters from the job itself before running the test